packages/docs/docs/medications/formulary.md

```diff
@@ -0,0 +1,298 @@
+
+Other metadata fields to note include `packaging`, `doseForm`, and `intendedRoute`.
+
+- The `packaging` attribute represents the type of packaging and quantity of the drug in the package. For instance, `'{tbl}'` is used to indicate a tablet. More details can be found in the [MedicationKnowledge Package Type Codes](http://hl7.org/fhir/R4/valueset-medicationknowledge-package-type.html) and the guide to [units of measure](https://terminology.hl7.org/4.0.0/ValueSet-v3-UnitsOfMeasureCaseSensitive.html).
+- The `doseForm` attribute is used to indicate the physical form of the medication, for example, whether it is a liquid, powder, pill, etc. The [SNOMED Form Codes](http://hl7.org/fhir/R4/valueset-medication-form-codes.html) can be used for reference.
+- The `intendedRoute` attribute suggests how the drug is to be ingested, whether orally, intravenously, etc. The [SNOMED Route Codes](http://hl7.org/fhir/R4/valueset-route-codes.html) are available for more information.
+
+We will be providing a comprehensive example of a [`MedicationKnowledge`](/docs/api/fhir/resources/medicationknowledge) resource that incorporates drugCharacteristics, packaging, doseForm, and intendedRoute.
+
+:::note A note about units
+
+All units for medication quantities are [UCUM units of measure](https://terminology.hl7.org/4.0.0/ValueSet-v3-UnitsOfMeasureCaseSensitive.html). This includes standard SI units, as well as healthcare specific unit codes. A common unit for medications is `{tbl}`, which stands for "tablets", which is useful for quantifying the amount of medication in a package.
+
+:::
+
+<details>
+<summary>
+Example Drug Characteristics
+</summary>
+
+```ts
+{
+  "resourceType": "MedicationKnowledge",
+  "code": {
+    "text": "acetaminophen 325 MG [Tylenol]",
+    "coding": [
+      {
+        "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+        "code": "569998"
+      },
+      {
+        "system": "http://hl7.org/fhir/sid/ndc",
+        "code": "5058049501"
+      }
+    ]
+  },
+  "drugCharacteristic": [
+    {
+      "type": {
+        "text": "Color",
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/medicationknowledge-characteristic",
+            "code": "color"
+          }
+        ]
+      },
+      "valueString": "white"
+    },
+    {
+      "type": {
+        "text": "Imprint Code",
+        "coding": [
+          {
+            "system": "http://terminology.hl7.org/CodeSystem/medicationknowledge-characteristic",
+            "code": "imprintcd"
+          }
+        ]
+      },
+      "valueString": "Tylenol"
+    }
+  ],
+  "amount": {
+    "value": 100,
+    "unit": "{tbl}"
```

Comment:
Is this a standard unit?

We should add `system` property here to clarify

Reply:
It's a UCUM unit, which is "standard" within FHIR. I'll add the system string